target/linux/ramips/dts/mt7620a_lb-link_bl-w1200.dts

```diff
@@ -0,0 +1,190 @@
+/*
+ * Device Tree file for the LB-Link BL-W1200
+ * based on Edimax BR-6478AC V2
+ *
+ * Copyright (C) 2016 Rohan Murch <rohan.murch@gmail.com>
+ * Copyright (C) 2016 Hans Ulli Kroll <ulli.kroll@googlemail.com>
+ * Copyright (C) 2017 James McKenzie <openwrt@madingley.org>
+ * Copyright (C) 2020 Pawel Dembicki <paweldembicki@gmail.com>
+ *
+ * This file is licensed under the terms of the GNU General Public
+ * License version 2.  This program is licensed "as is" without any
+ * warranty of any kind, whether express or implied.
+ */
```

Comment:
Please use SPDX license identifier instead.
I don't know whether the "based on" comment is really necessary, if yes, please put that info into the commit message and remove it here.